google/cloud/security/inventory/pipelines/load_group_members_pipeline.py

```diff
@@ -85,14 +86,14 @@ def _transform(self, groups_members_map):
                        'member_email': member.get('email'),
                        'raw_member': json.dumps(member)}
 
-    def _retrieve(self):
-        """Retrieve the membership for a given GSuite group.
+    def _retrieve(self, group_ids):  # pylint: disable=arguments-differ
```

Comment:
Argh, probably didn't commit correctly.  Will catch-up on these, and just fix it now that it's merged.